tests/e2e/e2e_test.go

```diff
@@ -0,0 +1,373 @@
+package e2e
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// Hardcoding the location of kedge, which is in root of project directory
+var KedgeLoc = "kedge"
+var ProjectPath = "$GOPATH/src/github.com/kedgeproject/kedge/"
```

Comment:
I still don't get it.. What's the point of this then? The tests will use the binary located at $GOPATH not the root directory `./kedge`.